jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessor.java

```diff
@@ -147,7 +147,14 @@ public static JibBuildRunner createJibBuildRunnerForRegistryImage(
         inferredAuthProvider,
         rawConfiguration.getToCredHelper().orElse(null));
 
+    boolean isAlwaysCacheBaseImage =
+        Boolean.getBoolean(
```

Comment:
`Boolean.getBoolean()` doesn't do what you think. Probably `Boolean.valueOf()`.

Reply:
Right, it was looking for SystemProperty.